src/AtmosphereModels/set_atmosphere_model.jl

```diff
@@ -130,8 +130,6 @@ end
         θ = potential_temperature[i, j, k]
     end
 
-    g = thermo.gravitational_acceleration
-    z = znode(i, j, k, grid, c, c, c)
```

Comment:
These lines also got removed in another merged commit: 4e642d144cee2203aa9c0a5f2662d0e106b12c7d.  Was that done on purpose?